yt/data_objects/static_output.py

```diff
@@ -1456,6 +1456,37 @@ def define_unit(self, symbol, value, tex_repr=None, offset=None, prefixable=Fals
         _define_unit(self.unit_registry, symbol, value, tex_repr=tex_repr,
                      offset=offset, prefixable=prefixable)
 
+    def rotate_frame_by(self, shift_angle, deg=False):
+        """
+        (experimental)
+        Shift azimuth coordinate 'theta' by *shift_angle*, counter-clockwise, and in radians by default.
+        This method is supported for polar and cylindrical geometries, not for spherical.
+
+        Parameters
+        ----------
+        shift_angle : float
+            angle by which coordinates should be shifted
+        deg : bool, optional
+            set to True to use a *shift_angle* in degrees
+        """
+        rotating_axes = {"polar": 1, "cylindrical": 2}
+
+        if self.geometry not in rotating_axes.keys():
+            raise ValueError("Unsupported geometry '%s'." % self.geometry)
+
+        theta_axis = rotating_axes[self.geometry]
+
+        if self.dimensionality < theta_axis - 1:
+            raise ValueError("Unsupported dimensionality '%s' with geometry '%s'." % (self.dimensionality, self.geometry))
+
+        if deg:
+            shift_angle = np.deg2rad(shift_angle)
+
+        theta_edges = [self.index.grid_left_edge[:, theta_axis], self.index.grid_right_edge[:, theta_axis]]
```

Comment:
Just to check, this is a list, so when you iterate a few lines below, it's just over a list of two items, right?  The left and right edges?

Reply:
Exactly !